.github/workflows/macos.yml

```
@@ -46,11 +46,13 @@ jobs:
         uses: ammaraskar/gcc-problem-matcher@0.2.0
 
       - name: Build (${{ matrix.build-type }})
+        env:
+            CMAKE_PREFIX_PATH: ${{ matrix.qt-version-major == '6'}} && ${{ env.Qt6_Dir }} || ${{ env.Qt5_Dir }}
         run: |
           mkdir build
           cd build
-          qmake -config ${{ matrix.build-type }} ../src/Notes.pro
-          make -j$(sysctl -n hw.logicalcpu)
+          cmake .. -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -DCMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH}"
+          make -j$(nproc)
```

```suggestion
          cmake --build build-dir
```

P.S.: `nproc` doesn't exist on macOS.

If you want parallelism while building, you should export `CMAKE_BUILD_PARALLEL_LEVEL=$(sysctl -n hw.logicalcpu)`